Test/Extensions/CertInfoHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace fido2_net_lib;
+
+internal static class CertInfoHelper
+{
+    public static byte[] CreateCertInfo(
+        ReadOnlySpan<byte> magic,
+        ReadOnlySpan<byte> type,
+        ReadOnlySpan<byte> qualifiedSigner,
+        ReadOnlySpan<byte> extraData,
+        ReadOnlySpan<byte> clock,
+        ReadOnlySpan<byte> resetCount,
+        ReadOnlySpan<byte> restartCount,
+        ReadOnlySpan<byte> safe,
+        ReadOnlySpan<byte> firmwareRevision,
+        ReadOnlySpan<byte> tPM2BName,
+        ReadOnlySpan<byte> attestedQualifiedNameBuffer)
+    {
+        var stream = new MemoryStream();
```

Comment:
Do we need a using here?

Reply:
Technically, MemoryStreams don't need disposed -- as they're backed by managed memory that can be safely garbage collected. However, I like adding using statements to them anyway to match other stream types. Will update.

Here's the official documentation here:

```
This type implements the [IDisposable](https://learn.microsoft.com/en-us/dotnet/api/system.idisposable?view=net-7.0) interface, but does not actually have any resources to dispose. This means that disposing it by directly calling [Dispose()](https://learn.microsoft.com/en-us/dotnet/api/system.idisposable.dispose?view=net-7.0#system-idisposable-dispose) or by using a language construct such as using (in C#) or Using (in Visual Basic) is not necessary.
```

https://learn.microsoft.com/en-us/dotnet/api/system.io.memorystream?view=net-7.0

Reply:
@iamcarbon Oh, TIL. I was taught to always dispose streams and never try to be clever, which was a good lesson but overly cautious.